components/FourYearPlanPage.tsx

```diff
@@ -67,17 +67,23 @@ export const FourYearPlanPage: FC<FourYearPlanType> = memo(
     const defaultInformationType = "Requirements (Calculated)"; // The default
     const [informationTypes, setInformationTypes] = useState<string[]>([defaultInformationType]);
     const [displayedInformationType, setDisplayedInformationType] = useState<string | undefined>(defaultInformationType);
-
+    console.log(semesters);
     function initializeSemesters(): SemesterType[] {
       const tempSemesters = [];
+      let year: number = 0;
       for (let i = 0; i < semestersLength; i++) {
+        if (i % 2 === 0) {
+          year++;
```

Comment:
Not a big deal, but this function could use a comment describing what specifically it's initializing the semesters as (e.g. creates semesters for four years with 2 semesters per year, one being fall the other spring)